app/helpers/events_helper.rb

```diff
@@ -39,4 +39,12 @@ def canceled_replacement_event_label(event, event_schedule, *label_classes)
       content_tag :span, 'REPLACEMENT', class: (['label', 'label-info'] + label_classes)
     end
   end
+
+  def track_selector_input(form)
+    if @program.tracks.any?
+      form.input :track_id, as: :select,
+                            collection: @program.tracks.where(state: 'confirmed', cfp_active: true).pluck(:name, :id),
```

Comment:
I didn't have the scopes when I initially wrote that code, and then I forgot about it